frontend/src/styles/vars.scss

```diff
@@ -176,6 +176,7 @@ $_lifecycle_dormant: $_danger;
     --radius: 0.375rem;
     --radius-sm: 0.25rem;
     --shadow-elevation: 0px 16px 16px -16px rgb(0 0 0 / 35%);
+    --modal-shadow-elevation: 0px 16px 16px -16px rgb(0 0 0 / 35%);
```

Comment:
I feel like for modals we should be using the same shadow as for other floating stuff, i.e. popovers/dropdowns. I agree the current shadow does not necessary convey that the element is floating above other content, since it looks button-like. @corywatilo You recently mentioned improving dropdowns, maybe you have some opinions on this?